lib/treatments.js

```diff
@@ -16,7 +16,7 @@ function storage (env, ctx) {
     obj.created_at = created_at.toISOString();
 
     var preBolusCarbs = '';
-    if (obj.preBolus > 0 && obj.carbs) {
+    if (obj.preBolus != 0 && obj.carbs) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/800382157/issues/source?bid=2452379&fileBranchId=2725437#l19)